core/libs/models/src/file_metadata.rs

```diff
@@ -70,6 +70,18 @@ impl FileMetadata for EncryptedFileMetadata {
     fn deleted(&self) -> bool {
         self.deleted
     }
+    fn display(&self) -> String {
+        match self.file_type() {
+            FileType::Folder => format!("{}{}", self.id, "/"),
```

Comment:
minor detail I think we should make it clear that these are ids, also perhaps slighly simpler way to write this: 
```suggestion
            FileType::Folder => format!("id: {}/", self.id, "/"),
```